README.md

```diff
@@ -11,4 +11,19 @@ The whole dataset will be loaded into the Fedora repository <repo.url>, or view
 
 To load your own dataset, convert it to jcr/xml format and place the data files under src/main/resources/data directory, then run "mvn -Dfcrepo.url=&lt;repo.url&gt; exec:java" to load it to the repository.
 
+- - -
+## WebAC resources
```

Comment:
An ASCII art graphical depiction of the resources and ACLs that are being created wold be helpful.
...also possibly accompanied by reference to the various use cases.